### PR TITLE
Use dynamic choices in migrations

### PR DESCRIPTION
**Pull request by bhrutledge**
Fix for #32, based on suggestions from @jsma.

The tests are passing, but I haven't yet confirmed that this resolves the issue in the django CMS project where it was discovered. I'll check that in the next few hours.

### CODE REVIEW

File: djangocms_style/migrations/0006_removed_null_fields.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from __future__ import unicode_literals
 
 from django.db import migrations, models
-
+from djangocms_style.models import CLASS_CHOICES
```
Comment (FinalAngel) [inline djangocms_style/migrations/0006_removed_null_fields.py]: there seems to be a space missing

**Comment by bhrutledge** [comment]
@FinalAngel Done. I wasn't so bold as to bump the version number. 😄

**Comment by FinalAngel** [comment]
@bhrutledge great work, no worries I'll handle that :)

**Comment by bhrutledge** [comment]
@FinalAngel thanks! I confirmed that this resolves the issue in my project.

**Comment by FinalAngel** [comment]
@bhrutledge perfect merging and testing, You can expect a release somewhen early next week

**Pull request merged**